packages/grid/x-data-grid/package.json

```diff
@@ -3,19 +3,17 @@
   "version": "5.5.1",
   "description": "The community edition of the data grid component (MUI X).",
   "author": "MUI Team",
-  "main": "build/index-cjs.js",
-  "module": "build/index-esm.js",
-  "types": "build/x-data-grid.d.ts",
+  "main": "src/index.ts",
```

Comment:
```suggestion
  "main": "src/index.js",
```

Reply:
Why ? Pre-build there is no `src/index.js`
The build process changes those fields, the `package.json` released has a `main` and a `module` pointing to various bundles

Reply:
If it will add the correct extension during build, then there're other `package.json`s pointing to a non-existing `index.js`, e.g. `packages/grid/x-data-grid-pro/package.json`.

Reply:
Indeed, fixed for `x-data-grid-pro`, `x-data-grid-generator` and `x-license-pro`